src/validators/auth.ts

```diff
@@ -112,3 +113,46 @@ export async function validateRefreshToken(
     }
   }
 }
+
+/**
+ * check for existing user
+ *
+ * @param  {Request}   req
+ * @param  {Response}   res
+ * @param  {NextFunction} next
+ * @returns {Promise}
+ */
+export async function checkUser(
+  req: Request,
+  res: Response,
+  next: NextFunction
+) {
+  try {
+    const mobile = req.body.mobile;
+    const email = req.body.email;
```

Comment:
Let's destructure it.
```suggestion
    const {
           body: {
                     mobile, email
           }
    }
    const email = req.body.email;
```

Reply:
didnt get what you are doing

Reply:
It's destructing the res object, taking the body out of it and again destructing it to take the mobile and email out of it.

Reply:
res refers to response or something else?
It should be req, haina?

Reply:
done